sage_modabvar/abvar.py

```diff
@@ -2022,6 +2054,44 @@ def _ambient_hecke_matrix_on_modular_symbols(self, n):
         self.__ambient_hecke_matrix_on_modular_symbols[n] = T
         return T
 
+    def frobenius_polynomial(self, p):
+        """
+        Computes the frobenius polynomial at `p`. If this abelian variety is
+        not simple, a ValueError will be raised.
+
+        INPUT:
+
+        - ``p`` -- prime number
+
+        OUTPUT:
+            a monic integral polynomial
+
+        EXAMPLES::
+            from sage_modabvar import AbelianVariety
```

Comment:
Missing `sage:` prompt at the beginning of the line.  Did you run the tests?  Did they work with the missing prompt?